### PR TITLE
Just read the code... 

### PR DESCRIPTION
**Pull request by ThePythonGuy3**

### CODE REVIEW

**Comment by Anuken** [comment]
well.
1. this doesn't compile
2. I **guarantee** that someone would report this as a bug when it happens

**Comment by markozajc** [comment]
ahahah, 109 but automatic

**Comment by ThePythonGuy3** [comment]
Okay then. Would you like it if it said "Midnustry?" (and it compiled)

**Comment by ThePythonGuy3** [comment]
@Anuken Now it works, and it isnt just plain ohno. It says Minudstry

**Comment by ThePythonGuy3** [comment]
I cant pack sprites for some reason so i just uploaded the sprite to assets-raw/sprites/ui

File: core/src/mindustry/ui/fragments/MenuFragment.java [hunk 1]
```diff
@@ -85,8 +91,11 @@ public void build(Group parent){
             float fy = (int)(Core.graphics.getHeight() - 6 - logoh) + logoh / 2 - (Core.graphics.isPortrait() ? Scl.scl(30f) : 0f);
 
             Draw.color();
```
Comment (RandomTNT) [inline core/src/mindustry/ui/fragments/MenuFragment.java:93]: is this colouring text 
line

File: core/src/mindustry/ui/fragments/MenuFragment.java [hunk 2]
```diff
@@ -17,6 +17,10 @@
 import mindustry.graphics.*;
 import mindustry.ui.*;
 
+import java.time.LocalDateTime;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
 import static mindustry.Vars.*;
```
Comment (RandomTNT) [inline core/src/mindustry/ui/fragments/MenuFragment.java:24]: Importing  libs

File: core/src/mindustry/ui/fragments/MenuFragment.java [hunk 3]
```diff
@@ -17,6 +17,10 @@
 import mindustry.graphics.*;
 import mindustry.ui.*;
 
+import java.time.LocalDateTime;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
```
Comment (RandomTNT) [inline core/src/mindustry/ui/fragments/MenuFragment.java:22]: calendar libs

File: core/src/mindustry/ui/fragments/MenuFragment.java [hunk 4]
```diff
@@ -17,6 +17,10 @@
 import mindustry.graphics.*;
 import mindustry.ui.*;
```
Comment (RandomTNT) [inline core/src/mindustry/ui/fragments/MenuFragment.java:18]: basic libs

**Comment by RandomTNT** [comment]
comments

File: core/src/mindustry/ui/fragments/MenuFragment.java (hunk 3 above)
Comment (ThePythonGuy3) [reply core/src/mindustry/ui/fragments/MenuFragment.java:22]: Apparently intellij added some packaged from my tests. And i dont need them anymore :/

**Comment by ThePythonGuy3** [comment]
What are the comments you wrote for

**Comment by Anuken** [comment]
...I'd rather do this myself when the time comes instead of making it automatic.

**Comment by ThePythonGuy3** [comment]
Ohno

**Pull request closed without merging**